index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+   
+    <link rel="stylesheet" type="text/css" href="stylemb.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+    <div class="flex_container">
+
+        <header >
+         <bar class="barstatuslight"> 
+           
+                    <timeblock class="timeblock"  >
+                        <img src="img/TimeStyle.png" alt="9.13">
+                    </timeblock>
+                    <Group3 class ="Group3">
+                        <img src="img/Group3.png" alt="Network">
+                    </Group3>
+            </bar>
+        </header>
+
+            <nav  >              
+           
+            <compactbar class="compactbar">
+                <toobardark class="toobardark">
+                   
+                    <normalbutton class="normalbutton">
+                        <icons class="icons">
+
+                            <unions class="unions">
+                                <img src="img/Vector.png" alt="Network">
+                                <vector1 class="vector1">
+
+                                </vector1>
+                                <vector2 class="vector2">
+                                    
+                                </vector2>
+                                <vector3 class="vector3">
+                                    
+                                </vector3>
+                            </unions>
+
+                        </icons>
+
+                    </normalbutton>
+
+                </toobardark>
+                <logoplaceholder class ="logoplaceholder">
+                    <img src="img/logoplaceholder.png" alt="Network"> 
+
+                </logoplaceholder>
+
+            </compactbar>
+        </nav>  
```

Comment:
- [ ] [REQUIRED] Good job so far! Your `hamburger` button should be inside of the `header` tag 🙏🏼